### DISCUSSION TITLE
assigning tasks to users not on Github Enterprise - ghost user?

### DISCUSSION THREAD

**Discussion by Mike1794** [community/community#58254] [Enterprise]
Hi all,

Looking for suggestions of how to best-use Github enterprise for a specific task:

We're using an Enterprise instance of Github at my company. On my project, i'm specifically using it as a task/issue management tool, for a project that's mostly hardware-based.   I know that's an unusual use-case for Github, but it seems to work well.

I'm looking for ideas on how to bookkeep tasks assigned to people (or outsource companies) that don't (or can't) have accounts in our system.    Like a 'ghost user' or users that i (or someone else) can assign things to, and type updates from, so i know who's working on what.      I've thought about using 'Labels' but i'm not sure that's a great way to do it.   I'm just currently been adding updates myself, but that means everything is assigned to me.

**Comment by the-officialjosh** [comment]
Using GitHub Enterprise as a task/issue management tool for hardware-based projects is a creative approach. To bookkeep tasks assigned to people or external entities without GitHub accounts, here are a few suggestions:

1. Use a dedicated user account: Create a generic user account within your GitHub Enterprise instance that represents the external users or outsourced companies. Assign tasks to this account and use it for tracking updates. You can mention the responsible person or company in the task description or comments to indicate who is actually working on it.

2. Utilize project boards: Set up project boards within GitHub to visualize and track the progress of tasks. You can create custom columns to represent different stages or assignees. In this case, you can create a column for external entities and move tasks to their respective columns to indicate progress.

3. Implement issue templates: Create issue templates tailored for tasks assigned to external entities. These templates can include specific fields to capture important information about the task, such as the assigned person or company. By using these templates consistently, you can ensure proper bookkeeping of the tasks.

4. Leverage labels and milestones: Along with assigning tasks to a dedicated user account, you can utilize labels and milestones to further organize and categorize the tasks. Create labels specific to external entities or outsourced companies and apply them to the corresponding tasks. Milestones can be used to group related tasks together.

Remember, these suggestions provide a workaround for tracking tasks assigned to non-GitHub users within the GitHub platform. However, it's worth considering a dedicated project management tool if you require more advanced features specifically designed for managing tasks and assignments in a collaborative environment.

**Comment by Mike1794** [comment]
Thanks for the suggestions.

I should have been clearer, the project is currently at the 'debugging' phase, that's why we're not using a dedicated project management tool, and why Github (in particular Zenhub) seems to work very well.   People can open issues, and we can prioritize, and assign them to other people to solve.

I've asked my admin about setting up a ghost user, and i got ghosted.....lol

i'll look into 2 and 3.

thanks!

**Reply by the-officialjosh** [reply]
Ok great I think I understand better now, I will like to know if any of these worked so I could use them when helping others with similar issues or better still mark that as the answer when it works
